general/administration/installing.md

```diff
@@ -27,25 +32,59 @@ The Jellyfin Docker image is available on [Docker Hub](https://hub.docker.com/r/
     `--volume /path/to/cache:/cache \`  
     `--volume /path/to/media:/media \`  
     `--net=host \`  
-    `jellyfin/jellyfin`  
+    `jellyfin/jellyfin:latest`  
```

Comment:
This change is unnecessary; `latest` is always the default tag when no tag is specified.
Please remove.

Reply:
I made the update as to be explicit to what version someone may be running rather than be implicit. Removing.